### PR TITLE
i18n(de): Add guides/rss translation

### PR DESCRIPTION
**Pull request by nbyx**
#### What kind of changes does this PR include?
- Translated content

#### Description
Add translation for guides/rss.md

### CODE REVIEW

File: src/pages/de/guides/rss.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+```bash
+# npm
+npm i @astrojs/rss
+# yarn
+yarn add @astrojs/rss
+# pnpm
+pnpm i @astrojs/rss
+```
+
+Stelle dann sicher, dass du in deiner `astro.config` eine [`site` konfigurierst](/de/reference/configuration-reference/#site). Damit kannst du [über die `SITE`-Umgebungsvariable](/de/guides/environment-variables/#default-environment-variables) Links in deinem RSS-Feed erzeugen.
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:24]:
```suggestion
Stelle dann sicher, dass du in deiner `astro.config` eine [`site` konfigurierst](/de/reference/configuration-reference/#site). Dies ermöglicht die Verwendung der [`SITE`-Umgebungsvariable](/de/guides/environment-variables/#default-environment-variables) zur Erzeugung von Links in deinem RSS-Feed.
```

File: src/pages/de/guides/rss.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+:::note[Benötigt v1]
+Die `SITE` Umgebungsvariable existiert nur in der neusten Astro 1.0 beta. Entweder aktualisiere auf die aktuellste Version von Astro (`astro@latest`), oder schreibe deine `site` manuell, wenn das nicht möglich ist (siehe Beispiele unten).
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:27]:
```suggestion
Die `SITE`-Umgebungsvariable existiert nur in der neusten Astro 1.0 Beta. Aktualisiere entweder auf die aktuelle Astro-Version (`astro@latest`) oder fülle den `site`-Parameter manuell, falls dies nicht möglich ist (siehe Beispiele unten).
```

File: src/pages/de/guides/rss.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+:::
+
+Lass und jetzt unser erstes RSS-Feed generieren! Erstelle eine `rss.xml.js`-Datei in deinem `src/pages/`-Ordner. `rss.xml` wird die Ausgabe-URL sein. Du kannst sie also nach deinen Wünschen anpassen, wenn du möchtest.
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:30]:
```suggestion
Lass uns jetzt unseren ersten RSS-Feed generieren! Erstelle die Datei `rss.xml.js` in deinem `src/pages/`-Ordner, um einen über die URL `rss.xml` erreichbaren Feed zu erzeugen. Du kannst den Dateinamen nach deinen Wünschen anpassen, wenn du eine andere Ausgabe-URL bevorzugst.
```

File: src/pages/de/guides/rss.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+Danach, importiere die `rss`-Hilfsfunktion aus dem `@astrojs/rss`-Paket und rufe sie mit folgenden Parametern auf:
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:32]:
```suggestion
Importiere nun die `rss`-Hilfsfunktion aus dem `@astrojs/rss`-Paket und rufe sie mit folgenden Parametern auf:
```

File: src/pages/de/guides/rss.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+```js
+// src/pages/rss.xml.js
+import rss from '@astrojs/rss';
+
+export const get = () => rss({
+    // `<title>` Feld in der XML-Ausgabe
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:39]:
```suggestion
    // `<title>`-Feld in der XML-Ausgabe
```

File: src/pages/de/guides/rss.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+    title: 'Buzz’s Blog',
+    // `<description>` Feld in der XML-Ausgabe
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:41]:
```suggestion
    // `<description>`-Feld in der XML-Ausgabe
```

File: src/pages/de/guides/rss.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+    description: 'Ein bescheidener Astronaut und sein Weg zu den Sternen',
+    // Basis-URL für RSS-<item>-Links
+    // SITE verwendet "site" aus der astro.config deines Projekts.
+    site: import.meta.env.SITE,
+    // Liste mit `<item>` in der XML-Ausgabe
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:46]:
```suggestion
    // Liste von `<item>`-Elementen in der XML-Ausgabe
```

File: src/pages/de/guides/rss.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+    // Einfaches Beispiel: Items für jede md-Datei in /src/pages erzeugen
+    // Siehe im Abschnitt "items erzeugen" für erforderliche Frontmatter und erweiterte Anwendungsfälle
+    items: import.meta.glob('./**/*.md'),
+    // (optional) benutzerdefiniertes xml einfügen
+    customData: `<language>en-us</language>`,
+  });
+```
+
+## `items` generieren
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:55]:
```suggestion
## Generieren von `items`
```

File: src/pages/de/guides/rss.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+    // `<title>` Feld in der XML-Ausgabe
+    title: 'Buzz’s Blog',
+    // `<description>` Feld in der XML-Ausgabe
+    description: 'Ein bescheidener Astronaut und sein Weg zu den Sternen',
+    // Basis-URL für RSS-<item>-Links
+    // SITE verwendet "site" aus der astro.config deines Projekts.
+    site: import.meta.env.SITE,
+    // Liste mit `<item>` in der XML-Ausgabe
+    // Einfaches Beispiel: Items für jede md-Datei in /src/pages erzeugen
+    // Siehe im Abschnitt "items erzeugen" für erforderliche Frontmatter und erweiterte Anwendungsfälle
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:48]:
```suggestion
    // Siehe Abschnitt "Generieren von `items`" für erforderliche Frontmatter und erweiterte Anwendungsfälle
```

File: src/pages/de/guides/rss.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+    items: import.meta.glob('./**/*.md'),
+    // (optional) benutzerdefiniertes xml einfügen
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:50]:
```suggestion
    // (optional) Benutzerdefinierten XML-Code einfügen
```

File: src/pages/de/guides/rss.md [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+    customData: `<language>en-us</language>`,
+  });
+```
+
+## `items` generieren
+
+Das `items`-Feld akzeptiert entweder:
+1. [Ein `import.meta.glob(...)` Ergebnis](#1-importmetaglob-ergebnis) **(nutze dies nur für `.md`-Dateien in  deinem `src/pages/`-Verzeichnis!)**
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:58]:
```suggestion
1. [Ein `import.meta.glob(...)`-Ergebnis](#1-importmetaglob-ergebnis) **(nutze dies nur für `.md`-Dateien in deinem `src/pages/`-Verzeichnis!)**
```

File: src/pages/de/guides/rss.md [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+2. [Liste der RSS-Feed-Objekte](#2-liste-der-rss-feed-objekte), jeweils mit einem `link`, `title`, `pubDate`, und optionalen `description` und `customData` Feldern.
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:59]:
```suggestion
2. [Eine Liste der RSS-Feed-Objekte](#2-liste-der-rss-feed-objekte), jeweils mit den Feldern `link`, `title` und `pubDate` sowie optional auch `description` und `customData`.
```

File: src/pages/de/guides/rss.md [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+
+### 1. `import.meta.glob`-Ergebnis
+
+Wir empfehlen diese Option als eine bequeme Abkürzung für `.md`-Dateien unter `src/pages/`. Jeder Eintrag sollte einen `title`, `pubDate` und optional `description` oder `customData` Felder in seinem Frontmatter haben. Wenn das nicht möglich ist, oder du es bevorzugst deinen Frontmatter im Code selbst zu generieren, [siehe Option 2](#2-liste-der-rss-feed-objekte).
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:63]:
```suggestion
Wir empfehlen diese Option als eine bequeme Abkürzung für `.md`-Dateien unter `src/pages/`. Jeder Eintrag sollte die Frontmatter-Eigenschaften `title` und `pubDate` sowie optional auch `description` und `customData` haben. Falls dies nicht möglich ist, oder du die Werte lieber selbst im Code generierst, [sieh dir Option 2 an](#2-liste-der-rss-feed-objekte).
```

File: src/pages/de/guides/rss.md [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+
+export const get = () => rss({
+    title: 'Buzz’s Blog',
+    description: 'Ein bescheidener Astronaut und sein Weg zu den Sternen',
+    site: import.meta.env.SITE,
+    items: import.meta.glob('./blog/**/*.md'),
+  });
+```
+
+Siehe die [Vite glob-import-Dokumentation](https://vitejs.dev/guide/features.html#glob-import) für weitere Informationen über diese Syntax.
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:79]:
```suggestion
Sieh dir die [Vite-Dokumentation zu Glob Import](https://vitejs.dev/guide/features.html#glob-import) für weitere Informationen über diese Syntax an.
```

File: src/pages/de/guides/rss.md [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+
+### 2. Liste der RSS-Feed-Objekte
+
+Wir empfehlen diese Option für `.md`-Dateien außerhalb des `pages`-Verzeichnises. Dies ist überlich beim Generieren von Routen [via `getStaticPaths`](/de/reference/api-reference/#getstaticpaths).
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:83]:
```suggestion
Wir empfehlen diese Option für `.md`-Dateien außerhalb des `pages`-Verzeichnises. Dies ist üblich beim Generieren von Routen [via `getStaticPaths`](/de/reference/api-reference/#getstaticpaths).
```

File: src/pages/de/guides/rss.md [hunk 16]
```diff
@@ -0,0 +1,120 @@
...
+
+Angenommen, deine `.md`-Einträge sind in einem `src/posts/`-Verzeichnis gespeichert. Jeder Eintrag hat ein `title`, `pubDate` und `slug` in seinem Frontmatter, wobei `slug` der Ausgabe-URL deiner Website entspricht. Wir können dann ein RSS-Feed mithilfe von [Vite's `import.meta.glob` helper](https://vitejs.dev/guide/features.html#glob-import) wie folgt generieren:
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:85]:
```suggestion
Angenommen, deine `.md`-Einträge sind in einem `src/posts/`-Verzeichnis gespeichert. Jeder Eintrag hat einen `title`, `pubDate` und `slug` in seinem Frontmatter, wobei `slug` der Ausgabe-URL auf deiner Website entspricht. Wir können dann wie folgt einen RSS-Feed mithilfe von [Vites Hilfsfunktion `import.meta.glob`](https://vitejs.dev/guide/features.html#glob-import) generieren:
```

File: src/pages/de/guides/rss.md [hunk 17]
```diff
@@ -0,0 +1,120 @@
...
+      link: post.url,
+      title: post.frontmatter.title,
+      pubDate: post.frontmatter.pubDate,
+    }))
+  });
+```
+
+## Ein Stylesheet hinzufügen
+
+Du kannst deinen RSS-Feed so gestalten, dass er für den Nutzer noch angenehmer wird, wenn er ihn in seinem Browser ansieht.
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:108]:
```suggestion
Du kannst deinen RSS-Feed ansprechend gestalten, um die Nutzererfahrung bei der Betrachtung im Browser zu verbessern.
```

File: src/pages/de/guides/rss.md [hunk 18]
```diff
@@ -0,0 +1,120 @@
...
+
+Nutze die Option `stylesheet`der `rss`-Funktion, um einen absoluten Pfad zu deinem Stylesheet anzugeben.
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:110]:
```suggestion
Nutze die Option `stylesheet` der `rss`-Funktion, um einen absoluten Pfad zu deinem Stylesheet anzugeben.
```

File: src/pages/de/guides/rss.md [hunk 19]
```diff
@@ -0,0 +1,120 @@
...
+
+```js
+rss({
+  // Nutze z.B. dein Stylesheet aus "public/rss/styles.xsl"
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:114]:
```suggestion
  // Beispiel: Nutze dein Stylesheet aus "public/rss/styles.xsl"
```

File: src/pages/de/guides/rss.md [hunk 20]
```diff
@@ -0,0 +1,120 @@
...
+  stylesheet: '/rss/styles.xsl',
+  // ...
+});
+```
+
+Falls du kein RSS-Stylesheet zur Verfügung hast, empfehlen wir das [Pretty Feed v3 Standard-Stylesheet](https://github.com/genmon/aboutfeeds/blob/main/tools/pretty-feed-v3.xsl), das du dir von GitHub herunterladen und in dein `public/`-Verzeichnis deines Projekts speichern kannst.
```
Comment (hippotastic) [inline src/pages/de/guides/rss.md:120]:
```suggestion
Falls du kein RSS-Stylesheet zur Verfügung hast, empfehlen wir das [Pretty Feed v3 Standard-Stylesheet](https://github.com/genmon/aboutfeeds/blob/main/tools/pretty-feed-v3.xsl), das du dir von GitHub herunterladen und im `public/`-Verzeichnis deines Projekts speichern kannst.
```

**Comment by nbyx** [comment]
Thanks again for the improvements. I applied all of them to the translation :)

**Comment by hippotastic** [comment]
LGTM!

**Pull request merged**